packages/jest-snapshot/src/inline_snapshots.ts

```diff
@@ -120,6 +120,13 @@ const groupSnapshotsByFile = groupSnapshotsBy(({frame: {file}}) => file);
 
 const indent = (snapshot: string, numIndents: number, indentation: string) => {
   const lines = snapshot.split('\n');
+  if (
+    lines.length > 2 &&
```

Comment:
Why is it `> 2`, not `> 1`?

Reply:
AFAIK, the inline snapshot will always be 1 line (single-line) or at least three lines since the last parathesis will always wrap to a new line if not single line snapshots.

```
      expect(42).toMatchInlineSnapshot(`42`);
      expect('A very long string that will wrap to the next line').toMatchInlineSnapshot(
        `"A very long string that will wrap to the next line"`
      );
      expect('\n').toMatchInlineSnapshot(`
        "
        "
      `);
```
But maybe there is some corner-case with 2 lines inline snapshot so I'll updated the code. Also added a comment describing the purpose of the code block.

Reply:
Yeah thought about that as well. Agreed that it probably makes no difference in practice, but I think it's good to go with the option that's most intuitive 👍